vtools/cli.py

```diff
@@ -31,11 +31,26 @@
               help="Sample(s) in positive-vcf to consider. "
                    "May be called multiple times",
               required=True)
-def evaluate_cli(call_vcf, positive_vcf, call_samples, positive_samples):
+@click.option("-s", "--stats", type=click.Path(writable=True),
+              help="Path to output stats json file", default='-')
+@click.option("-dc", "--discordant", type=click.Path(writable=True),
+              help="Path to output discordant VCF file",
+              required=False)
+def evaluate_cli(call_vcf, positive_vcf, call_samples, positive_samples, stats,
+                 discordant):
     c_vcf = VCF(call_vcf, gts012=True)
     p_vcf = VCF(positive_vcf, gts012=True)
-    evaluated = site_concordancy(c_vcf, p_vcf, call_samples, positive_samples)
-    print(json.dumps(evaluated))
+    st, disc = site_concordancy(c_vcf, p_vcf, call_samples,
+                                positive_samples)
+    # Write the stats json file
+    with click.open_file(stats, 'w') as fout:
+        print(json.dumps(st), file=fout)
+
+    # If specified, write the discordant variants
+    if discordant:
+        with click.open_file(discordant, 'w') as fout:
+            for record in disc:
+                print(record, file=fout, end='')
```

Comment:
fout.writeline could also work instead of print.

Reply:
The problem with using fout.write here is that `record` is not a string, but a `cyvcf2.cyvcf2.Variant`. So we would need to do something like `fout.write(str(record))`, which is not a lot nicer than using the current `print` statement. It's just unfortunate that converting a `record` to string adds a newline character, so we also have to specify `end=''` to the print statement.

Reply:
`print()` always adds a newline. `write()` should not.  `writeline()` also adds a newline.
If `str(record)` adds a newline already, then `fout.write(str(record))` should work as expected and is less strange (by convention) than using `print` to write to a file.

Reply:
But since it works, and it the code is totally readable I approved it :+1: